iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -36,7 +36,7 @@
     <Description>Device SDK for Azure IoT Hub</Description>
     <PackageIcon>nugetIcon.png</PackageIcon>
     <PackageProjectUrl>https://github.com/Azure/azure-iot-sdk-csharp</PackageProjectUrl>
-    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <Copyright>Â© Microsoft Corporation. All rights reserved.</Copyright>
```

Comment:
?

Reply:
No idea how that got in there ! it's generated by the script ... I will investigate